crates/optimism/flashblocks/src/service.rs

```diff
@@ -199,22 +220,24 @@ where
             }
         }
 
-        // advance new canonical message, if any to reset flashblock
-        {
+        if let Poll::Ready(state) = {
```

Comment:
we can use `Ok(state)` here, the error variant this returns here is just for indicated if the broadcast stream lagged, which will never be the case so we can simply ignore this case